src/ducks/balance/AccountRowCaption.jsx

```diff
@@ -6,13 +6,15 @@ import Icon from 'cozy-ui/transpiled/react/Icon'
 import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 
 import flag from 'cozy-flags'
-import { isErrored } from 'utils/triggers'
+import { trigger as triggerLibs } from 'cozy-client/dist/models'
```

Comment:
nit: it's sounds weird to import from dist/

Reply:
I made the change, thanks.